packages/react-ui-core/src/DropDownMenu/__tests__/DropDownMenu-test.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import DropDownMenu from '../DropDownMenu'
+
+const theme = {
+  DropDownMenu: 'DropDownMenu',
+  DropDownMenu_ListItem: 'DropDownMenu_ListItem',
+  DropDownMenu_Selected: 'DropDownMenu_Selected',
+}
+
+const items = [1, 2, 3]
+
+describe('DropDownMenu', () => {
+  const setup = props => {
+    const wrapper = mount(
+      <DropDownMenu
+        theme={theme}
+        options={items}
+        {...props}
+      />,
+    )
+    return {
+      wrapper,
+    }
+  }
+
+  it('Should pass options to List component', () => {
+    const { wrapper } = setup({ theme, options: items })
+
+    expect(wrapper.find('[data-tid="dropdown-menu-list-item"]').length).toEqual(3)
+  })
+
+  it('Should perform on click funcationality ', () => {
```

Comment:
Spelling error on `funcationality`.  Can we also be more specific on what exactly you're testing? The wording here seems like it would be more of a `describe` statement than an `it`.